src/viz_utils.py

```diff
@@ -22,16 +21,23 @@ def create_geojson(polygons, classids, lookup, params):
         poly = np.array(poly)
         poly = poly[:, [1, 0]] * params["ds_factor"]
         poly = poly.tolist()
+            
+        geom = geojson.Polygon([poly], precision=2)
+        if not geom.is_valid:
+            print(f"Polygon {i}:{[poly]} is not valid, skipping...")
+            continue
         # poly.append(poly[0])
+        measurements = {classifications: 0 for classifications in CLASS_LABELS_PANNUKE.keys()}
```

Comment:
Can you use `lookup` for this, this way you dont have to check which kind of model somebody is running again (pannuke/lizard).
```suggestion
        measurements = {classifications: 0 for classifications in lookup.values()}
```